src/CMakeLists.txt

```diff
@@ -133,7 +133,8 @@ set_target_properties( exiv2lib_int PROPERTIES
     COMPILE_DEFINITIONS exiv2lib_EXPORTS
 )
 
-target_link_libraries(exiv2lib_int PRIVATE ZLIB::ZLIB)
+# NOTE: Cannot use target_link_libraries on OBJECT libraries with old versions of CMake
```

Comment:
Which CMake version are we stuck with due to  older distributions?  
Does it maybe make sense to move to a newer one, and simply have the older distributions download a newer version?

Reply:
On Debian:9 the default CMake version is 3.7 and on CentOS:8 it is 3.11. 

I am the first one I would like to use the latest CMake version possible, but in practice I think this would make more difficult for distributions to include the latest Exiv2 version (need to create patches, and so on). But being honest, I am just guessing, I do not know how distribution maintainers consume and generate binaries for each of the distribution packages. 

Most of the times (like this one) it is only about tiny details, and the quality of the CMake code does not get affected (too much). If we find a good excuse/reason to require a CMake higher version, we can always discuss it in a separate issue/PR 😁

Reply:
Sounds reasonable to me! It's always a trade-off :)   
As long as it's not impacting our CMake too much I think you are right, no reason to upgrade just yet :)